index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,900;1,100;1,300&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Harmattan&family=Roboto:ital,wght@0,900;1,100;1,300&display=swap"
+      rel="stylesheet"
+    />
+    <title>Gideon's Portfolio</title>
+    <script
+      src="https://kit.fontawesome.com/9947fbd823.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+
+  <body>
+    <header>
+      <a href="#" class="name">Gidiz Logo</a>
+      <i class="fa-solid fa-bars"></i>
+    </header>
+
+    <section class="main-mobile-section">
+      <div class="ellipse1"></div>
+      <div class="ellipse2"></div>
+
+      <h1>Gideon Dadi</h1>
+      <h3>Software Engineer</h3>
+
+      <p class="advert-text">
+        I can help you build a product, feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hestiate to contact me.
+      </p>
+
+      <hr class="vector4" />
+
+      <div class="tech-experience">
+        <ul class="language">
+          <li><h2>Language</h2></li>
+          <li>Javascript</li>
+          <li>Ruby</li>
+          <li>CSS</li>
+          <li>Html</li>
+        </ul>
+
+        <ul class="frameworks">
+          <li><h2>Frameworks</h2></li>
+          <li>Bootstrap</li>
+          <li>Ruby on Rails</li>
+          <li>RSPec</li>
+          <li>CapyBara</li>
+          <li>Selenium</li>
+        </ul>
+
+        <ul class="skills">
+          <li><h2>Skills</h2></li>
+          <li>CodeKit</li>
+          <li>GitHub</li>
+          <li>Codepen</li>
+          <li>Gitlab</li>
+          <li>Terminal</li>
+        </ul>
+
+        <hr class="vector5" />
+      </div>
```

Comment:
As the previous reviewer requested, please, kindly try to stick to the Figma template as much as possible.

Please, see my comment above about the Figma template.

- [ ] The font should be regular and not italic.
- [ ] There should be more spacing around the items.
- [ ] The layout should be better distributed among the columns.

Please, kindly see the screenshots below for references.

| Your version | Figma template |
| -- | -- |
| ![image](https://user-images.githubusercontent.com/81232315/154140338-74edb9a9-fd16-4e88-b8b6-a3041e6eee02.png) | ![image](https://user-images.githubusercontent.com/81232315/154140422-9ca93545-7080-4e12-b517-e34d0a4f5f46.png) |

Reply:
Thank you so much

Reply:
👍🏽 I can see that you have removed the font from italics and did justice to the spacings